src/sentry/integrations/vsts/integration.py

```diff
@@ -33,13 +34,29 @@
 )
 
 
-class VstsIntegration(Integration, VstsIssueSync):
+class VstsIntegration(Integration, RepositoryMixin, VstsIssueSync):
     logger = logging.getLogger('sentry.integrations')
 
     def __init__(self, *args, **kwargs):
         super(VstsIntegration, self).__init__(*args, **kwargs)
         self.default_identity = None
 
+    def reinstall(self):
+        self.reinstall_repositories()
+
+    def get_repositories(self):
+        try:
+            repos = self.get_client().get_repos(self.instance)
+        except ApiError:
+            repos = []
+        data = []
+        for repo in repos['value']:
+            data.append({
+                'name': '%s\%s' % (repo['project']['name'], repo['name']),
```

Comment:
Does VSTS represent it with a backslash?

Reply:
@EvanPurkhiser No it doesn't. VSTS doesn't represent it like this at all.... I just had no idea how to write it. If you or  @MeredithAnya have a better idea let me know. Otherwise I can just follow up with another pr. Probably shouldn't limit this one.